denonavr/ir_code.py

```diff
@@ -0,0 +1,507 @@
+ALL_POWER_ON_OFF = 'RCKSK0410001'
+ALL_POWER_ON = 'RCKSK0410002'
+ALL_POWER_OFF = 'RCKSK0410003'
+MAIN_ZONE_ON_OFF = 'RCKSK0410005'
+MAIN_ZONE_ON = 'RCKSK0410006'
+MAIN_ZONE_OFF = 'RCKSK0410007'
+TRIGGER_1_ON = 'RCKSK0410012'
+TRIGGER_1_OFF = 'RCKSK0410013'
+TRIGGER_2_ON = 'RCKSK0410014'
+TRIGGER_2_OFF = 'RCKSK0410015'
+KEY1 = 'RCKSK0410016'
+KEY2 = 'RCKSK0410017'
+KEY3 = 'RCKSK0410018'
+KEY4 = 'RCKSK0410019'
+KEY5 = 'RCKSK0410020'
+KEY6 = 'RCKSK0410021'
+KEY7 = 'RCKSK0410022'
+KEY8 = 'RCKSK0410023'
+KEY9 = 'RCKSK0410024'
+KEY0 = 'RCKSK0410025'
+CURSOR_UP = 'RCKSK0410027'
+CURSOR_DOWN = 'RCKSK0410028'
+CURSOR_LEFT = 'RCKSK0410029'
+CURSOR_RIGHT = 'RCKSK0410030'
+ENTER = 'RCKSK0410031'
+RETURN = 'RCKSK0410034'
+SETUP_MENU = 'RCKSK0410035'
+MEMORY = 'RCKSK0410036'
+UPGRADE_ID_ON = 'RCKSK0410042'
+Bilingual_Mode_toggle__JP_only = 'RCKSK0410045'
```

Comment:
Please convert all constant names to upper case that they fit to the python style guide